grader/src/main/java/edu/pdx/cs410J/grader/scoring/ProjectSubmissionOutputFileParser.java

```
@@ -1,27 +1,106 @@
+              String[] splitContent=fileContents.split(Pattern.quote("."));
+              projectSubmission.setProjectName(splitContent[splitContent.length -1]);
+              projectSubmission.setStudentId(splitContent[splitContent.length -2]);
+
+            }
+
+            if(fileContents.contains("Submitted by")){ //fileContents has StudentName
+             String [] splitContent=fileContents.split("Submitted by ");
+             projectSubmission.setStudentName(splitContent[splitContent.length -1]);
+            }
+
+            if(fileContents.contains("Submitted on")){ //fileContents has Submission Time
+                String [] splitContent=fileContents.split("Submitted on ");
+                projectSubmission.setSubmissionTime(parseSubmissionTime(splitContent[splitContent.length -1]));
+
+            }
+
+            if(fileContents.contains("Graded on")){ //fileContents has Graded Time
+                String [] splitContent=fileContents.split("Graded on");
+                projectSubmission.setGradedTime(parseGradingTime(splitContent[splitContent.length -1].trim()));
+            }
+              if(fileContents.contains("out of")){ //fileContents has Total Points and Awarded Points
+                  String [] splitContent=fileContents.split("out of");
+                  String keyword="out of";
+                  String awardedPoints=fileContents.substring(0, fileContents.indexOf(keyword));
+                  if(!awardedPoints.isEmpty() && awardedPoints != "") {
+                      double specifiedGrade = Double.parseDouble(awardedPoints);
+                      projectSubmission.setScore(specifiedGrade);
+                  }
+                  projectSubmission.setTotalPoints(Double.parseDouble(splitContent[splitContent.length -1].trim()));
+              }
+
+
+          }
+
+      } catch (IOException e) {
+          e.printStackTrace();
```

Please don't "swallow" this exception.  Instead, declare that this method `throws` an `IOException` and let the calling code deal with it.